aten/src/ATen/native/layer_norm.cpp

```diff
@@ -266,6 +269,28 @@ Tensor rms_norm_symint(
   const Tensor& weight = *weight_maybe_owned;
   _check_rms_norm_inputs_symint(input, normalized_shape, weight);
 
+#ifdef USE_MPS
+  if (weight_opt.has_value()) {
+    const Tensor weight = weight_opt.value();
+    const bool any_nested = input.is_nested() || weight.is_nested();
```

Comment:
Are nested tensors even defined for MPS?

Reply:
I followed a similar pattern as in SDPA: https://github.com/pytorch/pytorch/blob/057d9aff39c00aeccc5d5272e56a71849d418491/aten/src/ATen/native/transformers/attention.cpp#L755